### DISCUSSION TITLE
[New Device Support] ROKU

### DISCUSSION THREAD

**Discussion by ThatGuySM** [PRProd/HA-Firemote#38] [New Device Family Support Request]
Really love the work you put into this and how nice it looks! Really wish you could make it a tad more universal so we can make it work with ROKU too.

**Comment by PRProd** [comment]
@ThatGuy9811 - Thanks for your encouragement!  I put in a lot of work to make it look the way it does, and the only payment for my effort is the likes and kind words from Home Assistant community members, so I REALLY appreciate it when kind folks like you take the time to write notes like that.

As far as expanding the Firemote project to other devices, those thoughts have started to be considered on my end.  As the project's name might suggest, "Firemote" was named as such because its target has always been specifically on Amazon Fire devices.  However, Amazon Fire devices are just one flavor of Android media player devices, so I've read reports from some who have reached out directly or commented on other threads discussing their trial and error successes using this card with other Android (non Fire TV) devices.

Unfortunately, I don't think that Roku is based off of the Android platform.  I could be completely wrong about that though.  I haven't investigated it yet.  If that's the case, it would understandably take more effort to include a Roku than it would to include another family of Android devices.

I'm not sure if I'm ready to disclose the device I'm most thinking about including into the project this year yet, but perhaps this thread might be a decent place to compile a list of new/different device families that are NOT Amazon Fire devices ([devices not in this list](https://github.com/PRProd/HA-Firemote/wiki/Existing-Amazon-Devices---Support-Chart))

From the top of my head, here is start of a list of possible targets...
- Android Shield
- Xiaomi MI
- Roku

**Comment by PRProd** [comment]
@ThatGuy9811 - I created a poll so you and others could weigh in as well.  [#41 - Check it out](https://github.com/PRProd/HA-Firemote/discussions/41)

**Comment by PRProd** [comment]
The poll over at https://github.com/PRProd/HA-Firemote/discussions/41 has been reset once again after the latest release which added Chromecast support, which means that your previous vote has been removed.  If you're still interested in support for this one, I recommend going back and voting again.

**Comment by PRProd** [comment]
@ThatGuy9811 - Apple TV won the last round, so after that release the poll over at #41 has been reset once again.  If you're still interested in Firemote support for Roku, I recommend going back and voting again.

**Comment by PRProd** [comment]
@ThatGuy9811 - GOOD NEWS!

Roku just won the poll! :tada: I am in the process of adding Roku support to Firemote right now!

Keep any eye on [this discussion thread](https://github.com/PRProd/HA-Firemote/discussions/41#discussioncomment-7801488) for early updates, and information on when early testing can begin.

**Comment by PRProd** [comment]
If you didn't get the memo yet, Roku support is here!
